### DISCUSSION TITLE
My 7 Stellar Quest Lore NFT's for Sorobanathon

### DISCUSSION THREAD

**Discussion by silence48** [stellar/sorobanathon#125] [General]
I have created my 7 stellar quest lore themed NFT's.   
the links are to the actual nft's on the litemint marketplace.
I've included preview pictures for your enjoyment also.

First we have the prologue:
[SorobanathonPrologue](https://litemint.com/items/GBE7O6WQ5KL5X3R2ILTDTXBRKCMIOUSVSWJLZHW4OHRQVU6KT4GN673L/StQuPrologue)
![image](https://user-images.githubusercontent.com/35656692/206889436-cb6bf834-1aaf-406e-83ab-e38360dc5387.png)

Chapter 1 our questers are at the entrance of the temple.
[Squaktahune Temple](https://litemint.com/items/GDWANH6O477ACKPHPVZJTTNHPSHGLSVORIKTWTHBW3T4LMCP5DHLLXXC/SqLoreChap01)
![image](https://user-images.githubusercontent.com/35656692/206889470-62c32986-a086-412b-bf33-85f88210d6ca.png)

Chapter 2 our questers must cross a lavapit to meet Zebulon.
[The Lava Snakes](https://litemint.com/items/GBNSEEEORFOQ5JLGCMLVVIFXUGDQ4J37CVWZOGCWH2SA3OOVOWGAZNUH/SqLoreChap02)
![image](https://user-images.githubusercontent.com/35656692/206889487-6454dcc2-2a8f-40c9-bd2f-e6ee358a1746.png)

Chapter 3 our questers find a mysterious table
[the mysterious table](https://litemint.com/items/GBRK7FSRLM2F5UZ6XCPJQGBLK6BOLCT2RNPPG22R2FII6DGGFGDAITAH/SqLoreChap03)
![image](https://user-images.githubusercontent.com/35656692/206889508-7c8029b5-0d89-486b-87be-b7f76398dc3e.png)

Chapter 4 our questers find an alien device.
[the alien device](https://litemint.com/items/GDDIZLPOVNN363EHZIVF7SGJXXTNS4OPXDASHFOZO3H4LPREIS3AJGDA/SqLoreChap04)
![image](https://user-images.githubusercontent.com/35656692/206889528-9ab45d05-b6ca-4310-9ec9-98b7deac445e.png)

Chapter 5 our questers see the golden soroban and need to call for a ferry ride.
[The ferrymans phone](https://litemint.com/items/GB3I5WUJ24S52V4SK74AXQL7QPP66GHETNJHY37FP6ZYNA7MHBGDLTOK/SqLoreChap05)
![image](https://user-images.githubusercontent.com/35656692/206889559-e2a42dd7-5612-442f-aa6b-299260a68fa3.png)

Chapter 6 our questers finally find the golden soroban on the island.
[the golden soroban](https://litemint.com/items/GARBPHAV2QYAFGHWQXOT23WPSPTCKIG2BXYFTA4GT4TQN6UP37P4ZVTT/SqLoreChap06)
![image](https://user-images.githubusercontent.com/35656692/206889586-cbe8ac90-9806-450e-9be0-17b97e06823f.png)

once my listings are approved on litemint you can [view the whole collection here:](https://litemint.com/explore?t=%23sorobanathon&c=sorobanathon)

By the way I am silence#8008 on discord.

**Comment by briwylde08** [comment]
These are amazing

**Reply by silence48** [reply]
> These are amazing

Thank you!

**Comment by stellarjohn** [comment]
Hi Silence!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by silence48** [reply]
thanks 

> Hi Silence!
> 
> Congratulations! Your contribution to has been accepted! 🎉 Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
> 
> Please let us know if you have any questions.
> 
> Cheers! John

Thank you! I filled it out.
